vm/ubpf_vm.c

```diff
@@ -49,6 +49,8 @@ ubpf_create(void)
         return NULL;
     }
 
+    vm->regs = NULL;
```

Comment:
`vm->regs` is already `NULL` from the use of malloc, right?

Reply:
Good catch. I will fix that.